interop/configs/branch.json

```diff
@@ -0,0 +1,40 @@
+{
+  "scripts": {
```

Comment:
What about other combinations: with extensions and external tree etc. (there are 8 combinations)? Probably doesn't matter much, just for completeness.

Reply:
https://github.com/mlswg/mls-implementations/issues/155